frontend/javascripts/oxalis/controller/camera_controller.js

```diff
@@ -262,54 +241,60 @@ export function rotate3DViewTo(id: OrthoView, animate: boolean = true): void {
       [OrthoViews.PLANE_YZ]: [0, -1, 0],
       [OrthoViews.PLANE_XZ]: [0, 0, -1],
     };
-
-    to = {
-      dx: positionOffset[id][0],
-      dy: positionOffset[id][1],
-      dz: positionOffset[id][2],
-      upX: upVector[id][0],
-      upY: upVector[id][1],
-      upZ: upVector[id][2],
-      l,
-      t,
-      r: l + width,
-      b: t - height,
-    };
+    up = upVector[id];
+    position = [
+      positionOffset[id][0] + flycamPos[0],
+      positionOffset[id][1] + flycamPos[1],
+      positionOffset[id][2] + flycamPos[2],
+    ];
   }
+  // Ensure a valid width and height if camera ended up with no width and height due to a bug.
```

Comment:
Is this actually a bug or does this only happen exactly once when the camera is not initialized, yet?

Reply:
This happened during refactoring the rotation, but in the current code this should not happen anymore.

I simply removed it :+1: